SwiftyStoreKit/PaymentQueueController.swift

```diff
@@ -91,6 +94,26 @@ class PaymentQueueController: NSObject, SKPaymentTransactionObserver {
         paymentQueue.remove(self)
     }
 
+	var unhandledTransactions: [Purchase] {
+		guard let completeTransactions = completeTransactionsController.completeTransactions else {
+			print("SwiftyStoreKit.completeTransactions() should be called once when the app launches. Returning no results")
+			return []
+		}
+		let purchases = paymentQueue.transactions.flatMap { transaction -> Purchase? in
+			guard transaction.transactionState != .purchasing else {
+				return nil
+			}
+			return Purchase(
+				productId: transaction.payment.productIdentifier,
+				quantity: transaction.payment.quantity,
+				transaction: transaction,
+				originalTransaction: transaction.original,
+				needsFinishTransaction: !completeTransactions.atomically
```

Comment:
This method will return purchases/transactions for payments that have not yet been finished.

Eventually, these transactions will be handled by either `PaymentsController`, `RestorePurchasesController`, `CompleteTransactionsController`.

However, this code always assumes `CompleteTransactionsController` when deciding the value of `needsFinishTransaction`, which makes me wonder if `Purchase` is the right thing to return.

For your use case, what properties of `SKPaymentTransaction` do you actually need?